res/controllers/Roland_DJ-505-scripts.js

```diff
@@ -1405,6 +1412,58 @@ DJ505.CueLoopMode = function(deck, offset) {
 };
 DJ505.CueLoopMode.prototype = Object.create(components.ComponentContainer.prototype);
 
+DJ505.EditMode = function(deck, offset) {
+    components.ComponentContainer.call(this);
+    this.ledControl = DJ505.PadMode.HOTCUE;
+    this.color = DJ505.PadColor.APPLEGREEN;
+
+    this.pads = new components.ComponentContainer();
+    for (var i = 0; i <= 3; i++) {
+        var baseKey = [
+            "intro_start",
+            "intro_end",
+            "outro_start",
+            "outro_end",
+        ][i];
+        var color = (i % 2) ? DJ505.PadColor.AZURE : DJ505.PadColor.BLUE;
```

Comment:
```suggestion
        var color = (i > 1) ? DJ505.PadColor.AZURE : DJ505.PadColor.BLUE;
```
This might look better. It's up to you though.